test/instructions.cc

```diff
@@ -304,14 +304,15 @@ TEST(Instructions, validate_ramp_instructions) {
   test_instructions({VALHALLA_SOURCE_DIR
                      "test/pinpoints/instructions/ramp_take_toward_driving_side_right.pbf"},
                     expected_routes_size, expected_legs_size, expected_maneuvers_size, maneuver_index,
-                    "Take the ramp toward Pennsylvania Avenue.", "",
-                    "Take the ramp toward Pennsylvania Avenue.", "");
+                    "Take the PA 283 West ramp toward Harrisburg.",
+                    "Take the Pennsylvania 2 83 West ramp.",
+                    "Take the Pennsylvania 2 83 West ramp toward Harrisburg.");
```

Comment:
Why did this test change? Should we keep the old test in place and add a new one?

Reply:
The old test was just a turn channel onto Pennsylvania Ave - not a ramp - therefore, needed to change the test
![image](https://user-images.githubusercontent.com/7515853/75170014-75b61700-56f7-11ea-83ef-96b019ef51bd.png)
![image](https://user-images.githubusercontent.com/7515853/75169735-0dffcc00-56f7-11ea-8eb6-360f689173e3.png)